doc/plugins/eu.esdihumboldt.hale.doc.user/html/new/5_1_0.xhtml

```diff
@@ -82,6 +95,14 @@
 					</p>
 				</td>
 			</tr>
+			<tr id="update_inspire_schemas">
+				<td class="title">Update offline versions of INSPIRE schemas to 2024.1.2</td>
+				<td class="content">
+					<p>
+						We've updated the offline versions of INSPIRE schemas to 2024.1.2, utilizing the INSPIRE schema release v.2024.1.2 - 21/02/2022. Notable changes include the update of the [hy] schema to version 5.0, reflecting the update of the hy-p schema to version 5. Additionally, various schemas, such as [us-xx], have undergone typo fixes for improved accuracy.
```

Comment:
The "typo fixes" are not notable because they were introduced when creating 2024.1, so they were no issue before.

Not sure what you mean with improved accuracy?

There is quite a number of schemas where there is now a version 5.0, why mention HydroPhysicalWaters explicitly?

Reply:
I have taken I from this page of readme
https://github.com/INSPIRE-MIF/application-schemas/tree/main/schemas